### PR TITLE
docs: update sentence syntax, grammar, etc. (README.md)

### PR DESCRIPTION
**Pull request by chroventer**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -31,17 +31,15 @@ Download & install the Java runtime and download the Lavalink.jar file.
...
-- Create an `application.yml` file in your working directory and copy the [example](https://github.com/Frederikam/Lavalink/blob/master/LavalinkServer/application.yml.example) into the created file and edit it with your configuration.
-
-- Run the jar file by running `java -jar Lavalink.jar` in a Terminal window.
+- Create an `application.yml` file in your working directory and copy the [example](https://github.com/Frederikam/Lavalink/blob/master/LavalinkServer/application.yml.example) content into the created file (`application.yml`) and modify it with your configuration.
+- Run the `Lavalink.jar` file by running `java -jar Lavalink.jar` in your Terminal/cmd/Powershell.
 
 ## Example usage
 
-> Note: Discord.js is used in this example, but it does work with Eris with the same example.
+> Note: Erela.js does not work with Eris, thus we are using [Discord.js](https://discord.js.org) for this example.
```
Comment (Anish-Shobith) [inline readme.md:39]: It does work with eris

File: readme.md [hunk 2]
```diff
@@ -51,30 +49,30 @@ const nodes = [{
     host: "localhost",
     port: 2333,
     password: "youshallnotpass",
-}]
+}];
 
-// Ready event fires when the Discord.js client is ready.
-// Use once so it only fires once.
+// Ready event fires when the Discord.js client has sucessfulyl connected to the API.
```
Comment (Anish-Shobith) [inline readme.md:54]: typo

File: readme.md [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 [![License](https://badgen.net/github/license/WarHammer414/erela.js)](https://github.com/WarHammer414/erela.js/blob/master/LICENSE)
 
+> An easy-to-use Lavalink client for NodeJS.
+
 ## Documentation
 
 > Note: Some links do not work, I'll fix them when I can. The sidebar menu has the links that are broken.
 
-You can find the documentation at <http://projects.warhammer.codes/erelajs> (*this* link does work)
+New to Erela.js? Check out the [API documentation](http://projects.warhammer.codes/erelajs).
```
Comment (Anish-Shobith) [inline readme.md:15]: Maybe only documentation

**Comment by WarHammer414** [comment]
Erela will be moving to a plugin based system and won't support Discord.js or Eris natively, you'll need to install the necessary plugin to work with the client you're using. It will still work without a plugin but you'll need to listen for events to send and receive data to and from discords web-socket. There's additional information in the announcement channel in the Erela.js category.

Right now the readme is mostly correct so I'll be closing this PR. Once I finish the plugin system and both Discord.js and Eris plugins I'll update the readme, then you can change anything you see fit.

**Pull request closed without merging**